src/design-system/text-link/text-link.css.ts

```diff
@@ -33,7 +37,15 @@ export const container = style([
       },
     },
   },
-]);
+  variants: {
+    scheme: {
+      [TextLinkVariants.Inline]: sx({
+        px: '$8',
```

Comment:
Is it possible to whitelist `px` in copmonent's props and set it to `$8` by default, so that if you need 0 padding you just overwrite it? 

I have mixed feelings about this names `Inline` and `Standalone`. They are counterintuitive to me. I would expect that Inline has no paddings.

Reply:
If we would like to be really strict in how TextLink component may look, maybe we can allow only `$0` and `$8`, so we use default one, or reset to 0.

Reply:
Agreed, I will move this to pure props limited to the two values 👍